requirements.txt

```diff
@@ -2,7 +2,7 @@ commonmark==0.8.0
 gunicorn
 django-dotenv
 dj_database_url
-django
+django==2.2
```

Comment:
Pinning Django to version 2.2, since some of the code depends on functionality deprecated in later versions.

I've got plans to pin versions on more (all) of the packages, but I'll get to that later.